src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.module.scss

```diff
@@ -18,13 +18,24 @@
   margin-bottom: 1rem;
 }
 
+.keyStatTile:first-child:nth-last-child(4),
+.keyStatTile:first-child:nth-last-child(4) ~ .keyStatTile {
+  flex: 1 0 49%;
+  max-width: 49%;
```

Comment:
Don't think this should be here as it will make tiles only take up half the screen on mobile and tablet